### PR TITLE
add error handing for non blocking errors

### PR DESCRIPTION
**Pull request by Jlrine2**

### CODE REVIEW

**Comment by Jlrine2** [comment]
Before merging we should change hyp3 deployment so as not to clobor

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [PEP 440](https://www.python.org/dev/peps/pep-0440/) 
 and uses [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [o.o.1](https://github.com/ASFHyP3/hyp3-event-monitoring/compare/v0.0.1...v0.0.2)
```
Comment (asjohnston-asf) [inline CHANGELOG.md:9]: zero.zero.one

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
...
+### ADDED
```
Comment (asjohnston-asf) [inline CHANGELOG.md:10]: `Added`

Comment (asjohnston-asf) [reply CHANGELOG.md:10]: Or maybe `Changed`?

File: tests/test_find_new.py [hunk 3]
```diff
@@ -275,11 +297,14 @@ def test_submit_jobs_for_granule_bad_granule(tables):
...
     with patch('hyp3_sdk.asf_search.get_nearest_neighbors', lambda x: []):
-        find_new.submit_jobs_for_granule(hyp3, granule, event_id)
+        with pytest.raises(find_new.GranuleError)as e:
+            find_new.submit_jobs_for_granule(hyp3, event_id, granule)
+            assert type(e.__context__) == HyP3Error
 
-    products = tables.product_table.scan()['Items']
-    assert len(products) == 1
-    assert products[0]['status_code'] == 'FAILED'
+    responses.add(responses.GET, _BASELINE_API, status=400)
```
Comment (asjohnston-asf) [inline tests/test_find_new.py:304]: also need a test for a baseline `500` response?

File: find_new/src/find_new.py [hunk 4]
```diff
@@ -65,26 +81,23 @@ def submit_jobs_for_granule(hyp3, granule, event_id):
...
     granule_lists.append([granule])
 
-    neighbors = asf_search.get_nearest_neighbors(granule['granuleName'])
+    try:
+        neighbors = asf_search.get_nearest_neighbors(granule['granuleName'])
+    except requests.HTTPError as e:
+        if e.response.status_code in range(400, 500):
+            raise GranuleError()
+        if e.response.status_code >= 500:
+            print(f'unable to find pairs for {granule["granuleName"]} skipping...')
```
Comment (asjohnston-asf) [inline find_new/src/find_new.py:90]: should we log `str(e)` so we know the nature of the failure?
`Unable to find InSAR pairs for {}: {str(e)}`?

**Review by asjohnston-asf — APPROVED** [review]

**Pull request merged**